VAMobile/src/components/FormWrapper/FormFields/FormAttachments.test.tsx

```diff
@@ -51,55 +42,45 @@ context('FormAttachments', () => {
     mockNavigateToSpy = jest.fn()
     mockNavigationSpy.mockReturnValue(mockNavigateToSpy)
 
-    component = render(
+    render(
       <FormAttachments
         removeOnPress={removeOnPressSpy}
         largeButtonProps={{ label: 'add files', onPress: largeButtonSpy }}
         attachmentsList={attachments}
       />,
     )
-
-    testInstance = component.UNSAFE_root
   }
 
   beforeEach(() => {
     initializeTestInstance()
   })
 
-  it('initializes correctly', async () => {
-    expect(component).toBeTruthy()
-  })
-
   describe('when there are attachments', () => {
-    it('should display a remove link', async () => {
-      expect(testInstance.findAllByType(TextView)[2].props.children).toEqual('Remove')
+    it('should display a remove link', () => {
+      expect(screen.getAllByText('Remove').length).toBeGreaterThan(0)
```

Comment:
should be able to update this to getByRole('link', {name: 'Remove'})

Reply:
I can use `getAllByRole` but there's still more than one, so it has to use the "all" bit. Updated.